src/coreComponents/physicsSolvers/multiphysics/integratedTests/ExtendedDruckerPrager_Wellbore.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" ?>
```

Comment:
Ditto, we can have a base integrated test XML, with individual tests simply including the base specifications.  This will remove a ton of redundancy.  See other examples like this in the repo.

Reply:
Sure, will do it